new_contributors/example_article_submission.md

```diff
@@ -2,7 +2,7 @@
 
 title: A Brief History of Container Technology
 
-description: A brief history of container technology and how it has fundamentally altered the development of software today.
+A brief history of container technology and how it has fundamentally altered the development of software today.
```

Comment:
Can you revert this back? 

Don't edit files outside your article folder. If you find anything inaccurate in the documentation files, you can ping one of the managers on GitHub or Slack and they will make the necessary changes.

Reply:
I actually did not intend to change it, reverting it immediately.